.github/workflows/tests-r.yml

```diff
@@ -8,7 +8,7 @@ jobs:
       max-parallel: 4
       fail-fast: false
       matrix:
-        python-version: ['3.9']
+        python-version: ['3.12']
```

Comment:
why 3.12 in some places and 3.11 in others? should 3.11 be our standard latest especially in extras?

Reply:
I think that makes sense, updating